src/pcapi/routes/serialization/reimbursement_csv_serialize.py

```diff
@@ -7,6 +7,10 @@
 from pcapi.utils.date import MONTHS_IN_FRENCH
 
 
+def localeNumber(num):
```

Comment:
Proposition de de nommage (un peu) plus précise :

```suggestion
def format_number_as_french(num):
```